kong/db/migrations/core/012_213_to_220.lua

```diff
@@ -0,0 +1,25 @@
+return {
+  postgres = {
+    up = [[
+      CREATE TABLE IF NOT EXISTS "cluster_status" (
```

Comment:
Not to bike shed here, but does it make sense to call this `cluster_nodes` instead?

Reply:
What about `cluster_data_planes` and `/cluster/data_planes` respectively? We do use this term prominently in the config interface for Hybrid mode.